src/ms3/bs4_parser.py

```diff
@@ -2047,7 +2047,21 @@ def __init__(self, soup: bs4.BeautifulSoup, **logger_cfg):
         self.instrumentation_fields = ['longName', 'shortName', 'trackName', 'instrumentId', 'part_trackName']
         self.parsed_parts = ParsedParts(soup)
         self.soup_references_data = self.soup_references()  # store references to XML tags
-
+        self.INSTRUMENT_DEFAULTS = self.enlarge_instrument_defaults_keys()
```

Comment:
This is kind of an anti-pattern: INSTRUMENT_DEFAULTS is a class variable, meaning it can be retrieved via `Instrumentation.INSTRUMENT_DEFAULTS` even without instantiating an Instrumentation object. Also, the all-caps are conventionally used for a constant (i.e. unchanging value). Here, however, you change the constant upon initialization which may lead to inconsistent behaviour.

Why don't you simply create an additional field that maps other acceptable values to the relevant dict keys of INSTRUMENT_DEFAULTS? That also avoids duplicating the default dicts. Then we probably want a method like `get_default_config()` that checks the default key and, from that, the default configuration.

By the way, the default name should be the trackName, not the part_trackName, the former being more specific and the latter more general (and more arbitrary/readily user-defined, too). I've adapted the test cases accordingly.

Reply:
Created the method to map relevant keys in commit [c306a64](https://github.com/johentsch/ms3/pull/57/commits/c306a644b945a057606f4e2526716edffb07c79f)